app/controllers/new-signup.js

```diff
@@ -80,25 +134,64 @@ export default class NewSignUpController extends Controller {
       return (this.error = ERROR_MESSAGES.userName);
     }
 
-    registerUser(signupDetails)
-      .then((res) => {
-        if (res.status === 204) {
-          this.analytics.identifyUser();
-          this.analytics.trackEvent(NEW_SIGNUP_FLOW.USER_REGISTERED);
-          this.currentStep = this.LAST_STEP;
-        } else {
-          this.analytics.trackEvent(NEW_SIGNUP_FLOW.UNABLE_TO_SIGNUP);
+    if (this.dev) {
```

Comment:
Will `this.dev` be true for `URL?dev=false` or `?dev=anthing`?

Reply:
Done can you please check latest commit